src/FakeItEasy/Core/BootstrapperLocator.cs

```diff
@@ -24,10 +27,23 @@ public static IBootstrapper FindBootstrapper()
         {
             var bootstrapperInterface = typeof(IBootstrapper);
 
+#if FEATURE_REFLECTION_GETASSEMBLIES
             var appDomainAssemblies = AppDomain.CurrentDomain.GetAssemblies();
             var appDomainAssembliesReferencingFakeItEasy = appDomainAssemblies
                 .Where(assembly => !assembly.IsDynamic)
                 .Where(assembly => assembly.ReferencesFakeItEasy());
+#else
+            var fakeItEasyLibraryName = TypeCatalogue.FakeItEasyAssembly.GetName().Name;
+
+            var context = DependencyContext.Default;
+            var candidateLibraries = context.RuntimeLibraries.Where(
+                library => library.Dependencies.Any(
+                    dependency => string.Equals(dependency.Name, fakeItEasyLibraryName, StringComparison.Ordinal)));
+
+            var appDomainAssembliesReferencingFakeItEasy = candidateLibraries
+                .SelectMany(library => library.GetDefaultAssemblyNames(context))
+                .Select(Assembly.Load);
+#endif

```

Comment:
I am not savvy. Does this still exclude dynamic assemblies? Or is that not a concern in .NET Core?